src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -248,21 +248,21 @@ if (NGRAPH_CPU_ENABLE)
         set_property(TARGET cpu_backend APPEND PROPERTY LINK_FLAGS "-Wl,-Bsymbolic-functions -Wl,--exclude-libs=ALL")
     endif()
 
-    if (NGRAPH_MLIR_ENABLE)
-        # TODO: can we get away without LLVM/MLIR include path.
-        # Currently mlir backend compiler.hpp include LLVM/MLIR files
-        get_directory_property(MLIR_LLVM_INCLUDEPATH
-                               DIRECTORY ${NGRAPH_MLIR_SOURCE_DIR}
-                               DEFINITION MLIR_LLVM_INCLUDEPATH)
+    # if (NGRAPH_MLIR_ENABLE)
```

Comment:
clean up comments

Reply:
I want to re-enable MLIR in the CPU backend so I left this in. I did not want to make the compiler too complex so I figured a separate PR to fix the CPU backend would work. If you want I can add MLIR support back to the CPU backend in this PR.

Reply:
I think that would work best, that way this change is an addition and not a subtraction.